sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -102,19 +142,27 @@ src_test() {
 src_install() {
 	multibuild_foreach_variant multilib-minimal_src_install
 
+	if ! use system; then
 	(
 		shopt -s failglob || die "failglob failed"
 
 		# Make sure our man pages do not collide with glibc or man-pages.
-		for manpage in "${ED}"/usr/share/man/man3/crypt{,_r}.?*; do
+		for manpage in "${ED}$(get_xcmandir)"/man3/crypt{,_r}.?*; do
```

Comment:
This introduces double prefix (${ED} contains ${EPREFIX} and so does $(get_xcmandir) in the cross case).

Reply:
The issue here is, ED is defined to be `${EPREFIX}/${D}`. But `get_xcmandir` is guaranteed to return something prefixed with `${EPREFIX}`. 

As you've seen, it's no bother when `${EPREFIX}` is blank (most cases), but we can see it's wrong, so we should do something about it to help out Prefix users. The easiest fix here is to simply use `${D}`, not `${ED}` on this line.

(This is testable by setting up Gentoo Prefix but there's no need for you to do that)